Samples/Server/Opengl-SpinningCube/SpinningCubeServerGL.vcxproj

```diff
@@ -0,0 +1,125 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You've got a path naming issue here.  Case sensitive, Opengl puts the proj file in its own folder on non-windows systems.  Need to move this into OpenGL-SpinningCube.